pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -1221,27 +1221,8 @@ func (lbaas *LbaasV2) EnsureLoadBalancerDeleted(clusterName string, service *v1.
 		if lbSecGroup.Err != nil && !isNotFound(lbSecGroup.Err) {
```

Comment:
After deleting LB security group, the rules of LB security group will be deleted automatically.